### PR TITLE
Bento-1143

### PR DESCRIPTION
**Pull request by skiran86**
Bento-1143

- Add Support for special characters and foreign languages
- Bug Fix - comma and new line issue in cart comment section

### CODE REVIEW

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

File: src/pages/fileCentricCart/utils.js [hunk 1]
```diff
@@ -31,12 +31,17 @@ export function convertToCSV(jsonse, comments, keysToInclude, header) {
     let line = '';
     keysToInclude.map((keyName) => {
       if (line !== '') line += ',';
-      line += entry[keyName] !== null ? entry[keyName] : ' ';
+      let columnResult = entry[keyName];
```
Comment (AjayDoddapaneni) [inline src/pages/fileCentricCart/utils.js:34]: can we have same logic for the dashboard table csv download too

Comment (skiran86) [reply src/pages/fileCentricCart/utils.js:34]: Yes, It is included with https://github.com/CBIIT/bento-frontend/pull/307

Comment (AjayDoddapaneni) [reply src/pages/fileCentricCart/utils.js:34]: Okay 👍

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**